lib/list_versions.go

```diff
@@ -27,16 +27,17 @@ func GetTFList(mirrorURL string, preRelease bool) ([]string, error) {
 	var semver string
 	if preRelease == true {
 		// Getting versions from body; should return match /X.X.X-@/ where X is a number,@ is a word character between a-z or A-Z
-		semver = `\/(\d+\.\d+\.\d+)(-[a-zA-z]+\d*)?\/`
+		semver = `\/(\d+\.\d+\.\d+)(-[a-zA-z]+\d*)?\"`
```

Comment:
out of interest: what is the reason to quote double quote in regex? does it have any special meaning? or is this used down the code and is enclosed in double quotes? (the latter explain the need to quote)
also might probably need to adjust comment above this line to replace last char of regex.

Reply:
- escaping is not needed in [raw string literals](https://go.dev/ref/spec#:~:text=Unicode%20code%20point-,String%20literals,-A%20string%20literal) 
- could use word characters class \w for the '@' part
- we also could accomodate for possible reintroduction of trailing '/'

Not tested:

```suggestion
		semver = `/(\d+.\d+.\d+)(-\w+)?/?"`
```

Reply:
Without the double quote in the regex, I was finding that it was capturing both pre release and full release versions every time. The double quote acts as an ending delimiter so that the right versions are found when ignoring pre-release versions.

Reply:
Though the double quote doesn't need to be escaped, right?